LICENSE

```diff
@@ -0,0 +1,27 @@
+Copyright (c) 2018, Broad Institute, Inc., Verily Life Sciences LLC
```

Comment:
This is BSD-3 right? I think that's what we're supposed to use for DataBiosphere projects.

Reply:
🤷‍♂️ , I copied this directly from Leonardo